index.html

```diff
@@ -2659,7 +2659,7 @@ <h3>Working with file and path names</h3>
 			<li>The last two code points at the end of the Supplementary Planes (U+1FFFE, U+1FFFF … U+EFFFE, U+EFFFF)</li>
 		</ul></li>
 
-		<li>All Unicode <a href="https://www.unicode.org/Public/UCD/latest/ucd/PropList.txt">deprecated characters</a>.</li>
+		<li>All Unicode <a href="https://www.unicode.org/Public/UCD/latest/ucd/PropList.txt">deprecated characters</a> (search for "Deprecated" in the page).</li>
```

Comment:
Nit: the UCD isn't really a set of "pages", since they are just text files. Perhaps "... in the file" instead?

Reply:
Thanks. Fixed in https://github.com/w3c/bp-i18n-specdev/commit/1cb452facaa93cccf61401d5827d5cc077f68d68